extensions/qute/component/src/generated/resources/org/apache/camel/component/qute/qute.json

```diff
@@ -11,7 +11,7 @@
     "supportLevel": "Preview",
     "groupId": "org.apache.camel.quarkus",
     "artifactId": "camel-quarkus-qute-component",
-    "version": "3.2.2-SNAPSHOT",
+    "version": "3.2.3-SNAPSHOT",
```

Comment:
Just curious why this version does not be changed by the release script automatically since it is a genereated file?

Reply:
Same here. If someone remember what is special about qute, that would be great to share.
Even when setting the next version in [Camel Quarkus main branch](https://camel.apache.org/camel-quarkus/3.5.x/contributor-guide/release-guide.html#_next_version_in_camel_quarkus_main_branch), there are specific steps for qute.

Reply:
Probably because the `maven-release-plugin` is configured to only run `clean validate` as part of `preparationGoals` & `completionGoals`.

https://github.com/apache/camel-quarkus/blob/main/pom.xml#L402-L403

That won't be enough to run the component generation bits.